tests/unit/test_general_helpers.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022, Google LLC All rights reserved.
```

Comment:
Let's delete this file if it's not doing anything.

Reply:
It is necesssary for test coverage to do its thing. It is an empty test for an empty file (I don't know the history of that, just trying to fix a very broken build here).